test/cases/testdata/time/test-time-0970.yaml

```diff
@@ -0,0 +1,14 @@
+cases:
+- data:
+  modules:
+  - |
+    package generated
```

Comment:
[nit] This is fine, but since these are added manually, and no longer are generated, we could

- call the package something else (`package p`, `package test`, whatever you want)
- give our locals names (if you'd like)
- combine multiple cases into one file

Reply:
I have done some refactoring. What do you think of this test case?
```yaml
cases:
- data:
  modules:
  - |
    package test

    layout := "2006-01-02"

    diff = [minute_second, leap_year, not_leap_year, leap_year_one_day, different_tz] {
      minute_second := time.diff(time.now_ns()+61*1000*1000*1000, time.now_ns())
      leap_year := time.diff(time.parse_ns(layout, "2020-02-02"), time.parse_ns(layout, "2020-03-01"))
      not_leap_year := time.diff(time.parse_ns(layout, "2021-02-02"), time.parse_ns(layout, "2021-03-01"))
      leap_year_one_day := time.diff(time.parse_ns(layout, "2004-02-29"), time.parse_ns(layout, "2005-03-01"))
      different_tz := time.diff([time.now_ns()+60*1000*1000*1000, "UTC"], [time.now_ns(), "Local"])
    }
  note: time/diff
  query: data.test.diff = x
  want_result:
  - x:
    - [0,0,0,0,1,1]
    - [0,0,28,0,0,0]
    - [0,0,27,0,0,0]
    - [1,0,1,0,0,0]
    - [0,0,0,0,1,0]
 ```

Reply:
Or, any suggestion for how I can get multiple "want_result"'s. Since `query` is a string, I do not see any immediate solution for having multiple rules

Reply:
`query` can contain semicolons; also, I think you might get away with
```
query: data.test.diff = [minute_second, leap_year, not_leap_year, leap_year_one_day, different_tz]
```
? I haven't tried it, though, sorry

Reply:
I have tried a couple of different variants now, but I always get
```
exp: {{"different_tz": [0, 0, 0, 0, 1, 0]}, {"leap_year": [0, 0, 28, 0, 0, 0]}, {"leap_year_one_day": [1, 0, 1, 0, 0, 0]}, {"minute_second": [0, 0, 0, 0, 1, 1]}, {"not_leap_year": [0, 0, 27, 0, 0, 0]}}
got: {{"different_tz": [0, 0, 0, 0, 1, 0], "leap_year": [0, 0, 28, 0, 0, 0], "leap_year_one_day": [1, 0, 1, 0, 0, 0], "minute_second": [0, 0, 0, 0, 1, 1], "not_leap_year": [0, 0, 27, 0, 0, 0]}}
```

The latest variant I tried was this:
```yaml
cases:
- data:
  modules:
  - |
    package test

    layout := "2006-01-02"

    a = minute_second {
       minute_second := time.diff(time.now_ns()+61*1000*1000*1000, time.now_ns())
    }

    b = different_tz {
      different_tz := time.diff([time.now_ns()+60*1000*1000*1000, "UTC"], [time.now_ns(), "Local"])
    }

    c = leap_year {
      leap_year := time.diff(time.parse_ns(layout, "2020-02-02"), time.parse_ns(layout, "2020-03-01"))
    }

    d = not_leap_year {
      not_leap_year := time.diff(time.parse_ns(layout, "2021-02-02"), time.parse_ns(layout, "2021-03-01"))
    }

    e = leap_year_one_day {
      leap_year_one_day := time.diff(time.parse_ns(layout, "2004-02-29"), time.parse_ns(layout, "2005-03-01"))
    }

  note: time/diff
  query: >
    data.test.a = minute_second;
    data.test.b = different_tz;
    data.test.c = leap_year;
    data.test.d = not_leap_year;
    data.test.e = leap_year_one_day
  want_result:
  - minute_second: [0,0,0,0,1,1]
  - different_tz: [0,0,0,0,1,0]
  - leap_year: [0,0,28,0,0,0]
  - not_leap_year: [0,0,27,0,0,0]
  - leap_year_one_day: [1,0,1,0,0,0]
```

@srenatus Do you know why it expects the variables to be on this format `{{},{},{},...,{}}`? And possibly how I can overcome this?

Reply:
Try
```
  want_result:
  - minute_second: [0,0,0,0,1,1]
    different_tz: [0,0,0,0,1,0]
    leap_year: [0,0,28,0,0,0]
    not_leap_year: [0,0,27,0,0,0]
    leap_year_one_day: [1,0,1,0,0,0]
```